packages/styletron-react/src/index.js

```diff
@@ -420,6 +480,32 @@ will not recieve the provided engine instance. This scenario can arise, for exam
             elementProps.className = joined;
           }
 
+          if (__BROWSER__ && window.__STYLETRON_DEVTOOLS__) {
```

Comment:
Should be this also guarded by `&& __DEV__`? We probably don't want to ship this whole functionality with production bundles, right? Same for other places.

Reply:
Yep 👍 Good spot!